app/models/mainstream_browse_page.rb

```diff
@@ -10,6 +10,14 @@ class MainstreamBrowsePage
     to: :content_item
   )
 
+  FORMATS_TO_EXCLUDE = %w(
```

Comment:
These formats correspond to `format` in rummager, but we're comparing them with `schema_name` (to be `document_type`). We'll need to map them to the values in the content store (for example, `travel-advice` -> `travel_advice`).